src/main/lrsql/admin/routes.clj

```diff
@@ -169,13 +170,21 @@
                                      ri/delete-reaction)
      :route-name :lrsql.admin.reaction/delete]})
 
+
+
+
 (defn admin-lrs-management-routes [common-interceptors jwt-secret jwt-leeway no-val-opts]
   #{["/admin/agents" :delete (conj common-interceptors
                                    lm/validate-delete-actor-params
                                    (ji/validate-jwt jwt-secret jwt-leeway no-val-opts)
                                    ji/validate-jwt-account
                                    lm/delete-actor)
-     :route-name :lrsql.lrs-management/delete-actor]})
+     :route-name :lrsql.lrs-management/delete-actor]
+    ["/admin/openapi" :get  (conj common-interceptors
+                                   
+                                   (ji/validate-jwt jwt-secret jwt-leeway no-val-opts)
+                                   ji/validate-jwt-account
```

Comment:
Do we want the openapi endpoint to be authenticated?

Reply:
I thought about this---since they need auth to make any actual requests, didn't seem like it was actually denying anything useful.  Happy to open it up if we want.